dev/io.openliberty.org.jboss.resteasy.common.ee10/src/io/openliberty/restfulWS/utils/RestfulWSUtils.java

```diff
@@ -187,19 +186,20 @@ private static RESTfulEndpointLoggingIntrospector getRESTfulEndpointLoggingIntro
             @Override
             public RESTfulEndpointLoggingIntrospector run() {
                 try {
-                Bundle b = FrameworkUtil.getBundle(ResourceMethodRegistry.class);
-                if(b != null) {
-                    BundleContext bc = b.getBundleContext();
-                    ServiceReference<RESTfulEndpointLoggingIntrospector> sr = bc.getServiceReference(RESTfulEndpointLoggingIntrospector.class);
-                    return (RESTfulEndpointLoggingIntrospector)bc.getService(sr);
-                }
+                    Bundle b = FrameworkUtil.getBundle(ResourceMethodRegistry.class);
+                    if(b != null) {
+                        BundleContext bc = b.getBundleContext();
+                        ServiceReference<RESTfulEndpointLoggingIntrospector> sr = bc.getServiceReference(RESTfulEndpointLoggingIntrospector.class);
+                        return (RESTfulEndpointLoggingIntrospector)bc.getService(sr);
+                    }
                 } catch (NoClassDefFoundError ncdfe) {
                     // ignore - return null
                 }
                 return null;
             }});
     }
 
+    @FFDCIgnore(NoClassDefFoundError.class)
```

Comment:
This method doesn't catch NoClassDefFoundError either.  It is ClassNotFoundException in this method.